pkg/scheduler/framework/plugins/noderesources/balanced_allocation.go

```diff
@@ -24,6 +24,7 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/scheduler/framework"
+	plfeature "k8s.io/kubernetes/pkg/scheduler/framework/plugins/feature"
```

Comment:
since there is no chance of conflict with the other "feature" package (the point is not to import it), you don't need the rename/alias. You can do the same in all pkg/scheduler/framework but the registry.